### PR TITLE
Organize initial states

### PR DESCRIPTION
**Pull request by vitormattos**
I separated the content of complex methods in specific initial state values.

Making this I identified code executions that wasn't necessary. As example, the endpoint to display page to reset password.

### CODE REVIEW

File: lib/Service/SignFileService.php [hunk 1]
```diff
@@ -537,31 +537,17 @@ public function validateRenewSigner($uuid, $user): void {
 		$this->validateHelper->validateRenewSigner($uuid, $user);
 	}
 
-	public function getFileData(FileEntity $fileData, ?IUser $user, ?SignRequestEntity $signRequest = null): array {
-		$return['action'] = JSActions::ACTION_SIGN;
-		$return['sign'] = [
-			'uuid' => $fileData->getUuid(),
-			'filename' => $fileData->getName()
-		];
+	public function getSignerData(?IUser $user, ?SignRequestEntity $signRequest = null): array {
```
Comment (giovannism20) [inline lib/Service/SignFileService.php:540]:
```suggestion
	public function getSignerData(?IUser $user, ?SignRequestEntity $signRequest = null): array {
  $return = ['user' => ['name' => null]];

  if ($signRequest) {
    $return['user']['name'] = $signRequest->getDisplayName();
  } elseif ($user) {
    $return['user']['name'] = $user->getDisplayName();
  }
  
  return $return;
}
```

File: lib/Controller/PageController.php [hunk 2]
```diff
@@ -335,20 +313,16 @@ public function resetPassword(): TemplateResponse {
...
-			];
+			$this->initialState->provideInitialState('action', JSActions::ACTION_DO_NOTHING);
+			$this->initialState->provideInitialState('errors', [$this->l10n->t('Invalid UUID')]);
 		}
-		$this->initialState->provideInitialState('config', array_merge(
-			$config,
+		$this->initialState->provideInitialState('config',
 			$this->accountService->getConfig($this->userSession->getUser())
-		));
+		);
```
Comment (giovannism20) [inline lib/Controller/PageController.php:325]: Have one more closed parentheses here

File: lib/Controller/PageController.php [hunk 3]
```diff
@@ -306,19 +294,9 @@ public function validationFileWithShortUrl(): RedirectResponse {
 	#[PublicPage]
 	#[RequireSignRequestUuid]
 	public function resetPassword(): TemplateResponse {
-		$this->initialState->provideInitialState('config', array_merge(
+		$this->initialState->provideInitialState('config',
 			$this->accountService->getConfig($this->userSession->getUser()),
```
Comment (giovannism20) [inline lib/Controller/PageController.php:298]: Can remove the comma on final of the line

**Review by giovannism20 — CHANGES_REQUESTED** [review]

File: lib/Controller/PageController.php (hunk 3 above)
Comment (vitormattos) [reply lib/Controller/PageController.php:298]: At row 297? If yes, isn't possible. If are you talking about row 298, yes, is possible.

File: lib/Controller/PageController.php (hunk 2 above)
Comment (vitormattos) [reply lib/Controller/PageController.php:333]: was removed

File: lib/Controller/PageController.php (hunk 3 above)
Comment (giovannism20) [reply lib/Controller/PageController.php]: Yes, I was talking about 298 line.

**Review by giovannism20 — APPROVED** [review]

**Pull request merged**